util.py

```diff
@@ -23,15 +24,31 @@ def get_economics_status() -> Dict[str, Any]:
         status = {
             'latest_update': int(datetime.datetime.now().timestamp()),
             'backend': SERVER,
-            'total_supply': total_supply,
-            'circulating_supply': circulating_supply,
-            'mined_tokens': mined_tokens,
-            'released_premined_tokens': RELEASED_PREMINED_TOKENS
+            'total_supply':
+                get_decimals(total_supply, add_comma_separator) if decimals else total_supply,
+            'circulating_supply':
+                get_decimals(circulating_supply, add_comma_separator) if decimals else circulating_supply,
+            'mined_tokens':
+                get_decimals(mined_tokens, add_comma_separator) if decimals else mined_tokens,
+            'released_premined_tokens':
+                get_decimals(RELEASED_PREMINED_TOKENS, add_comma_separator) if decimals else RELEASED_PREMINED_TOKENS,
         }
 
     return status
 
 
+def get_decimals(value: int, add_comma_separator: bool) -> str:
+    assert sys.version_info >= (3, 6)
+    value /= 10**DECIMAL_PLACES
+    # This fixes the case where the decimal places are .00, so we must return with all zeros
+    string_formatter = '{:.{}f}'
+    if add_comma_separator:
+        # Adding comma as thousands separator to string formatter
+        string_formatter = string_formatter[:2] + ',' + string_formatter[2:]
```

Comment:
It feels too specific for the `string_formatter` on line 44. I'd just replace for `{:,.{}f}`.

Reply:
I need the two different formatters, depending if I want comma separator or not. I could have

```
string_formatter = '{:.{}f}'
string_formatter_with_comma = '{:,.{}f}'
```

Reply:
I wouldn't bother adding an alternative formatter (dots for thousands and comma for decimals). I don't get why this is useful, correct formatting should be a concern of the frontend (because there is so much more cases than just thousands and decimal separators). To me this API was just for people wanting to see the value easier, and just having `,` for thousands and `,` for decimals is enough.

Reply:
I got it. But isn't it easier to just set it? Instead of including a "," in that place? This `string_formatter[:2] + ',' + string_formatter[2:]` is harder to read w/out benefit.

Reply:
Ah, okay! I misunderstood your comment initially. Done